src/main/scala/com/gu/octopusthrift/models/OctopusArticle.scala

```diff
@@ -0,0 +1,108 @@
+package com.gu.octopusthrift.models
+
+import java.util.concurrent.TimeUnit
+
+import com.gu.flexibleoctopus.model.thrift._
+import org.joda.time.format.DateTimeFormat
+import org.joda.time.{ DateTime, DateTimeZone }
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+import scala.util.{ Success, Try }
+
+/**
+ * This model does not represent all fields sent by Octopus,
+ * only those that will be relevant in converting to our Thrift model
+ */
+case class OctopusArticle(
+  id: Int,
+  filename: String,
+  forPublication: String,
+  lawyered: String,
+  objectType: String,
+  objectNumber: Int,
+  lastModified: String,
+  inUseBy: Option[String],
+  isCheckedOut: String,
```

Comment:
The naming here suggests this should be a bool?

Reply:
Again, this is a representation of the raw data from Octopus. This value can be either `"Y"` or `"N"`. When we convert to Thrift, it becomes a bool. I suppose we could make that conversion earlier, as part of the JSON reads?

Reply:
> I suppose we could make that conversion earlier, as part of the JSON reads?

I think this depends on how we'd like to model it:
- An open case class to fit the raw input and a case class to fit the thriftable data and we catch issues during the conversion
- A strongly typed case class to fit the input and a case class to fit the thriftable data and we catch issues during the serialisation

Either works, I think. Happy to ship this and refactor later, should we need to.